monai/data/test_time_augmentation.py

```diff
@@ -109,6 +112,9 @@ def __init__(
         nearest_interp: bool = True,
         orig_meta_keys: Optional[str] = None,
         meta_key_postfix=DEFAULT_POST_FIX,
+        to_tensor: bool = True,
+        output_device: Union[str, torch.device] = "cpu",
+        post_func: Callable = lambda x: x,
```

Comment:
```suggestion
        post_func: Callable = _identity,
```

Reply:
Hi @rijobro ,

Sure, updated in the latest commit.

Thanks.